scep/scep_test.go

```diff
@@ -237,8 +247,9 @@ func newCSR(t *testing.T, priv *rsa.PrivateKey, email, country, cname string) []
 	return der
 }
 
-func loadTestFile(t *testing.T, path string) []byte {
+func readFile(t *testing.T, path string) []byte {
```

Comment:
It may be sweet if you were to consider that the path is always relative to the `testdata` directory and use something like `p := path.Join("testdata, relpath)`.

Reply:
Fixed in 97dd282993d59cc4fb08e88d33c34e90e0ce7a2b